soh/soh/GameMenuBar.cpp

```diff
@@ -201,12 +201,110 @@ namespace GameMenuBar {
+                            fpsSlider = 2;
+                        } else { // currentFps == 60
+                            fpsSlider = 3;
+                        }
+                    }
+                    if (CVarGetInteger("gMatchRefreshRate", 0)) {
+                        UIWidgets::DisableComponent(ImGui::GetStyle().Alpha * 0.5f);
+                    }
+
+                    if (ImGui::Button(" - ##WiiUFPS")) {
+                        fpsSlider--;
+                    }
+                    ImGui::SameLine();
+                    ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
+
+                    UIWidgets::Spacer(0);
+
+                    ImGui::SliderInt("##WiiUFPSSlider", &fpsSlider, 1, 3, "", ImGuiSliderFlags_AlwaysClamp);
+
+                    ImGui::SameLine();
+                    ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
+                    if (ImGui::Button(" + ##WiiUFPS")) {
+                        fpsSlider++;
+                    }
+
+                    if (CVarGetInteger("gMatchRefreshRate", 0)) {
+                        UIWidgets::ReEnableComponent("");
+                    }
+                    if (fpsSlider > 3) {
+                        fpsSlider = 3;
+                    } else if (fpsSlider < 1) {
+                        fpsSlider = 1;
+                    }
+
+                    if (fpsSlider == 1) {
+                        currentFps = 20;
+                    } else if (fpsSlider == 2) {
+                        currentFps = 30;
+                    } else if (fpsSlider == 3) {
+                        currentFps = 60;
+                    }
+                    CVarSetInteger("gInterpolationFPS", currentFps);
+                    SohImGui::RequestCvarSaveOnNextTick();
+                #else
+                    bool matchingRefreshRate =
+                        CVarGetInteger("gMatchRefreshRate", 0) && SohImGui::WindowBackend() != SohImGui::Backend::DX11;
+                    UIWidgets::PaddedEnhancementSliderInt(
+                        (currentFps == 20) ? "FPS: Original (20)" : "FPS: %d",
+                        "##FPSInterpolation", "gInterpolationFPS", minFps, maxFps, "", 20, true, true, false, matchingRefreshRate);
+                #endif
+                    UIWidgets::Tooltip(
+                        "Uses Matrix Interpolation to create extra frames, resulting in smoother graphics. This is purely "
+                        "visual and does not impact game logic, execution of glitches etc.\n\n"
+                        "A higher target FPS than your monitor's refresh rate will waste resources, and might give a worse result.\n\n"
+                        "For consistent input lag, set this value and your monitor's refresh rate to a multiple of 20."
+                    );
+                } // END FPS Slider
+
+                if (SohImGui::WindowBackend() == SohImGui::Backend::DX11) {
+                    ImGui::Dummy(ImVec2(0, 0));
```

Comment:
Can you change this into a `UIWidgets::Spacer(0)`?  We don't really need to keep using these Dummies since we've got our own wrapper for them.

Reply:
Must've forgotten temporarily I made that wrapper in the first place, because I used it in other places in this PR lol.